src/main/java/com/uid2/shared/Utils.java

```diff
@@ -191,4 +193,11 @@ public static InputStream convertHttpResponseToInputStream(HttpResponse<String>
 
         return new ByteArrayInputStream(responseBytes);
     }
+    public static MessageDigest createMessageDigest() {
```

Comment:
Maybe rename this function to make it specific to SHA-512?

Reply:
Renamed